src/fw/drivers/sf32lb52/jdi_lpm015m135a.c

```diff
@@ -149,18 +160,32 @@ void display_init(void) {
 
 void display_clear(void) {
   DisplayJDIState *state = DISPLAY->state;
+  uint8_t index = 0xFF;
 
   mutex_lock(s_update);
 
-  memset(s_framebuffer, 0xFF, DISPLAY_FRAMEBUFFER_BYTES);
+  while (index == 0xFF) {
+    psleep(10);
```

Comment:
I'm a bit confused with this part; shouldn't we always have a free buffer available? There can only be 1 transfer ongoing to the display.

Reply:
UI layer cannot write to the buffer being transferred to the display, am I missing something?